### ISSUE TITLE
HarfBuzz_jll.jl fails on Windows on latest Julia

### ISSUE THREAD

**Issue by ChrisRackauckas** [JuliaPackaging/Yggdrasil#6043]
```julia
julia> using HarfBuzz_jll
[ Info: Precompiling HarfBuzz_jll [2e76f6c2-a576-52d4-95c1-20adfe4de566]
ERROR: LoadError: InitError: could not load library "C:\Users\accou\.julia\artifacts\3aff78028a0bb2ad40ebc3ae7e465a52dcbe2730\bin\libcairo-2.dll"
The specified module could not be found.
Stacktrace:
  [1] dlopen(s::String, flags::UInt32; throw_error::Bool)
    @ Base.Libc.Libdl .\libdl.jl:117
  [2] dlopen(s::String, flags::UInt32)
    @ Base.Libc.Libdl .\libdl.jl:116
  [3] macro expansion
    @ C:\Users\accou\.julia\packages\JLLWrappers\QpMQW\src\products\library_generators.jl:54 [inlined]
  [4] __init__()
    @ Cairo_jll C:\Users\accou\.julia\packages\Cairo_jll\HW4xi\src\wrappers\x86_64-w64-mingw32.jl:18
  [5] _include_from_serialized(pkg::Base.PkgId, path::String, depmods::Vector{Any})
    @ Base .\loading.jl:831
  [6] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String, build_id::UInt64)
    @ Base .\loading.jl:1039
  [7] _require(pkg::Base.PkgId)
    @ Base .\loading.jl:1315
  [8] _require_prelocked(uuidkey::Base.PkgId)
    @ Base .\loading.jl:1200
  [9] macro expansion
    @ .\loading.jl:1180 [inlined]
 [10] macro expansion
    @ .\lock.jl:223 [inlined]
 [11] require(into::Module, mod::Symbol)
    @ Base .\loading.jl:1144
 [12] include(mod::Module, _path::String)
    @ Base .\Base.jl:419
 [13] top-level scope
    @ C:\Users\accou\.julia\packages\JLLWrappers\QpMQW\src\toplevel_generators.jl:188
 [14] include
    @ .\Base.jl:419 [inlined]
 [15] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt64}}, source::Nothing)     
    @ Base .\loading.jl:1554
 [16] top-level scope
    @ stdin:1
during initialization of module Cairo_jll
in expression starting at C:\Users\accou\.julia\packages\HarfBuzz_jll\zavaV\src\wrappers\x86_64-w64-mingw32.jl:4
in expression starting at C:\Users\accou\.julia\packages\HarfBuzz_jll\zavaV\src\HarfBuzz_jll.jl:2
in expression starting at stdin:1
ERROR: Failed to precompile HarfBuzz_jll [2e76f6c2-a576-52d4-95c1-20adfe4de566] to C:\Users\accou\.julia\compiled\v1.8\HarfBuzz_jll\jl_D8C2.tmp.
Stacktrace:
  [1] error(s::String)
    @ Base .\error.jl:35
  [2] compilecache(pkg::Base.PkgId, path::String, internal_stderr::IO, internal_stdout::IO, keep_loaded_modules::Bool)
    @ Base .\loading.jl:1707
  [3] compilecache
    @ .\loading.jl:1651 [inlined]
  [4] _require(pkg::Base.PkgId)
    @ Base .\loading.jl:1337
  [5] _require_prelocked(uuidkey::Base.PkgId)
    @ Base .\loading.jl:1200
  [6] macro expansion
    @ .\loading.jl:1180 [inlined]
  [7] macro expansion
    @ .\lock.jl:223 [inlined]
  [8] require(into::Module, mod::Symbol)
    @ Base .\loading.jl:1144
  [9] eval
    @ .\boot.jl:368 [inlined]
 [10] eval
    @ .\Base.jl:65 [inlined]
 [11] repleval(m::Module, code::Expr, #unused#::String)
    @ VSCodeServer c:\Users\accou\.vscode\extensions\julialang.language-julia-1.40.1\scripts\packages\VSCodeServer\src\repl.jl:222
 [12] (::VSCodeServer.var"#107#109"{Module, Expr, REPL.LineEditREPL, REPL.LineEdit.Prompt})()
    @ VSCodeServer c:\Users\accou\.vscode\extensions\julialang.language-julia-1.40.1\scripts\packages\VSCodeServer\src\repl.jl:186
 [13] with_logstate(f::Function, logstate::Any)
    @ Base.CoreLogging .\logging.jl:511
 [14] with_logger
    @ .\logging.jl:623 [inlined]
 [15] (::VSCodeServer.var"#106#108"{Module, Expr, REPL.LineEditREPL, REPL.LineEdit.Prompt})()
    @ VSCodeServer c:\Users\accou\.vscode\extensions\julialang.language-julia-1.40.1\scripts\packages\VSCodeServer\src\repl.jl:187
 [16] #invokelatest#2
    @ .\essentials.jl:729 [inlined]
 [17] invokelatest(::Any)
    @ Base .\essentials.jl:726
 [18] macro expansion
    @ c:\Users\accou\.vscode\extensions\julialang.language-julia-1.40.1\scripts\packages\VSCodeServer\src\eval.jl:34 [inlined]
 [19] (::VSCodeServer.var"#61#62")()
    @ VSCodeServer .\task.jl:484

(@v1.8) pkg> build HarfBuzz_jll

julia> using HarfBuzz_jll
[ Info: Precompiling HarfBuzz_jll [2e76f6c2-a576-52d4-95c1-20adfe4de566]
ERROR: LoadError: InitError: could not load library "C:\Users\accou\.julia\artifacts\3aff78028a0bb2ad40ebc3ae7e465a52dcbe2730\bin\libcairo-2.dll"
The specified module could not be found.
Stacktrace:
  [1] dlopen(s::String, flags::UInt32; throw_error::Bool)
    @ Base.Libc.Libdl .\libdl.jl:117
  [2] dlopen(s::String, flags::UInt32)
    @ Base.Libc.Libdl .\libdl.jl:116
  [3] macro expansion
    @ C:\Users\accou\.julia\packages\JLLWrappers\QpMQW\src\products\library_generators.jl:54 [inlined]
  [4] __init__()
    @ Cairo_jll C:\Users\accou\.julia\packages\Cairo_jll\HW4xi\src\wrappers\x86_64-w64-mingw32.jl:18
  [5] _include_from_serialized(pkg::Base.PkgId, path::String, depmods::Vector{Any})
    @ Base .\loading.jl:831
  [6] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String, build_id::UInt64)
    @ Base .\loading.jl:1039
  [7] _require(pkg::Base.PkgId)
    @ Base .\loading.jl:1315
  [8] _require_prelocked(uuidkey::Base.PkgId)
    @ Base .\loading.jl:1200
  [9] macro expansion
    @ .\loading.jl:1180 [inlined]
 [10] macro expansion
    @ .\lock.jl:223 [inlined]
 [11] require(into::Module, mod::Symbol)
    @ Base .\loading.jl:1144
 [12] include(mod::Module, _path::String)
    @ Base .\Base.jl:419
 [13] top-level scope
    @ C:\Users\accou\.julia\packages\JLLWrappers\QpMQW\src\toplevel_generators.jl:188
 [14] include
    @ .\Base.jl:419 [inlined]
 [15] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt64}}, source::Nothing)     
    @ Base .\loading.jl:1554
 [16] top-level scope
    @ stdin:1
during initialization of module Cairo_jll
in expression starting at C:\Users\accou\.julia\packages\HarfBuzz_jll\zavaV\src\wrappers\x86_64-w64-mingw32.jl:4
in expression starting at C:\Users\accou\.julia\packages\HarfBuzz_jll\zavaV\src\HarfBuzz_jll.jl:2
in expression starting at stdin:1
ERROR: Failed to precompile HarfBuzz_jll [2e76f6c2-a576-52d4-95c1-20adfe4de566] to C:\Users\accou\.julia\compiled\v1.8\HarfBuzz_jll\jl_D311.tmp.
Stacktrace:
  [1] error(s::String)
    @ Base .\error.jl:35
  [2] compilecache(pkg::Base.PkgId, path::String, internal_stderr::IO, internal_stdout::IO, keep_loaded_modules::Bool)
    @ Base .\loading.jl:1707
  [3] compilecache
    @ .\loading.jl:1651 [inlined]
  [4] _require(pkg::Base.PkgId)
    @ Base .\loading.jl:1337
  [5] _require_prelocked(uuidkey::Base.PkgId)
    @ Base .\loading.jl:1200
  [6] macro expansion
    @ .\loading.jl:1180 [inlined]
  [7] macro expansion
    @ .\lock.jl:223 [inlined]
  [8] require(into::Module, mod::Symbol)
    @ Base .\loading.jl:1144
  [9] eval
    @ .\boot.jl:368 [inlined]
 [10] eval
    @ .\Base.jl:65 [inlined]
 [11] repleval(m::Module, code::Expr, #unused#::String)
    @ VSCodeServer c:\Users\accou\.vscode\extensions\julialang.language-julia-1.40.1\scripts\packages\VSCodeServer\src\repl.jl:222
 [12] (::VSCodeServer.var"#107#109"{Module, Expr, REPL.LineEditREPL, REPL.LineEdit.Prompt})()
    @ VSCodeServer c:\Users\accou\.vscode\extensions\julialang.language-julia-1.40.1\scripts\packages\VSCodeServer\src\repl.jl:186
 [13] with_logstate(f::Function, logstate::Any)
    @ Base.CoreLogging .\logging.jl:511
 [14] with_logger
    @ .\logging.jl:623 [inlined]
 [15] (::VSCodeServer.var"#106#108"{Module, Expr, REPL.LineEditREPL, REPL.LineEdit.Prompt})()
    @ VSCodeServer c:\Users\accou\.vscode\extensions\julialang.language-julia-1.40.1\scripts\packages\VSCodeServer\src\repl.jl:187
 [16] #invokelatest#2
    @ .\essentials.jl:729 [inlined]
 [17] invokelatest(::Any)
    @ Base .\essentials.jl:726
 [18] macro expansion
    @ c:\Users\accou\.vscode\extensions\julialang.language-julia-1.40.1\scripts\packages\VSCodeServer\src\eval.jl:34 [inlined]
 [19] (::VSCodeServer.var"#61#62")()
    @ VSCodeServer .\task.jl:484
```

```julia
julia> versioninfo()
Julia Version 1.8.4
Commit 00177ebc4f (2022-12-23 21:32 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: 32 × AMD Ryzen 9 5950X 16-Core Processor
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-13.0.1 (ORCJIT, znver3)
  Threads: 1 on 32 virtual cores
Environment:
  JULIA_EDITOR = code
  JULIA_NUM_THREADS =
```

**Comment by ChrisRackauckas** [comment]
<img width="521" alt="Screenshot 2022-12-31 112809" src="https://user-images.githubusercontent.com/1814174/210149714-59115f46-6d75-400c-a8b3-08f40b4e535a.png">

And what is claimed to be missing exists.

**Comment by giordano** [comment]
See https://discourse.julialang.org/t/glmakie-and-plots-fail-to-precompile-under-julia-1-8-4/92087. The only work around at the moment is to add Julia to path, this will be fixed in 1.8.5 and 1.9+.

**Comment by giordano** [comment]
Should have been fixed by 1.8.5
